src/components/Configure/InstallIntegration.tsx

```diff
@@ -20,19 +20,35 @@ export interface MappedValue {
   mappedDisplayValue: string;
 }
 
-export type FieldMappingWithMappedValues = IntegrationFieldMapping & {
```

Comment:
I had to redeclare the fields of IntegrationFieldMapping since otherwise the exported type didn't show all the fields (unless we also exported IntegrationFieldMapping, which I don't want to do since it would be confusing)